content/facility.yaml

```
@@ -1,168 +1,153 @@
-  img: wellner
-  url: https://www.weller-tools.com/we1010na/
+  - name: Bantam PCB Mill
+    img: bantam
+    url: https://www.bantamtools.com/
 
-- name: Eventek KPS305D DC Power Supply
-  img: eventek
-  url: https://www.amazon.com/dp/B071RNT1CD
+  - name: Weller WE1010NA
+    img: wellner
+    url: https://www.weller-tools.com/we1010na/
 
-- name: Reflow Oven T962
-  img: t962
-  url: https://www.amazon.com/dp/B01LZYEF90
+  - name: Eventek KPS305D DC Power Supply
+    img: eventek
+    url: https://www.amazon.com/dp/B071RNT1CD
 
-- name: Andonstar AD407 Digital Microscope
-  img: andonstar
-  url: https://www.amazon.com/dp/B07VK52X9C
+  - name: Reflow Oven T962
+    img: t962
+    url: https://www.amazon.com/dp/B01LZYEF90
 
+  - name: Andonstar AD407 Digital Microscope
+    img: andonstar
+    url: https://www.amazon.com/dp/B07VK52X9C
 
 Kniiting:
-- name: Brother 930E Knitting Machine
-  img: brother
-
-- name: Pfaff Creative 4.5 Embroidery Machine
-  img: pfaff
-  url: http://www.pfaff.com/en-CA/Machines/creative-4-5
+  - name: Brother 930E Knitting Machine
+    img: brother
 
+  - name: Pfaff Creative 4.5 Embroidery Machine
+    img: pfaff
+    url: http://www.pfaff.com/en-CA/Machines/creative-4-5
 
 Power Tools:
-- name: Hitachi C10FCG
-  img: hitachi
-  url: https://www.amazon.com/dp/B07217ZVP5
-
-- name: WEN 4208 Drill Press
-  img: drill-press
-  url: https://www.amazon.com/dp/B00HQONFVE
+  - name: Hitachi C10FCG
+    img: hitachi
+    url: https://www.amazon.com/dp/B07217ZVP5
 
-- name: WEN 3959 Band Saw
-  img: band-saw
-  url: https://www.amazon.com/dp/B077QMBTLP
+  - name: WEN 4208 Drill Press
+    img: drill-press
+    url: https://www.amazon.com/dp/B00HQONFVE
 
-- name: Black+Decker 20V Drill
-  img: black-decker
-  url: https://www.amazon.com/dp/B00C625KVE
+  - name: WEN 3959 Band Saw
+    img: band-saw
+    url: https://www.amazon.com/dp/B077QMBTLP
 
+  - name: Black+Decker 20V Drill
+    img: black-decker
+    url: https://www.amazon.com/dp/B00C625KVE
 
 AR/VR:
-- name: Vicon Motion Capture
-  img: vicon
-  url: https://www.vicon.com/
+  - name: Vicon Motion Capture
+    img: vicon
+    url: https://www.vicon.com/
 
-- name: Oculus Quest
-  img: oculus-quest
-  url: https://www.oculus.com/quest/
+  - name: Oculus Quest
+    img: oculus-quest
+    url: https://www.oculus.com/quest/
 
-- name: Hololens 2
-  img: hololens-2
-  url: https://www.microsoft.com/en-us/hololens/
+  - name: Hololens 2
+    img: hololens-2
+    url: https://www.microsoft.com/en-us/hololens/
 
-- name: Hololens 1
-  img: hololens-1
-  url: https://docs.microsoft.com/en-us/hololens/hololens1-hardware
-
-- name: Azure Kinect DK
-  img: azure-kinect
-  url: https://azure.microsoft.com/en-us/services/kinect-dk/
+  - name: Hololens 1
+    img: hololens-1
+    url: https://docs.microsoft.com/en-us/hololens/hololens1-hardware
 
+  - name: Azure Kinect DK
+    img: azure-kinect
+    url: https://azure.microsoft.com/en-us/services/kinect-dk/
 
 Robotics:
```

Added Unitree Go2 Robot
```suggestion
Robotics:
  - name: Unitree Go2
    img: unitree-go2
    url: https://www.unitree.com/go2

```